### ISSUE TITLE
Embed `textarea` missing label

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-EmbedCodes#5]
The `textarea` used to display embed code is considered a form element, and it is currently missing a label. Consider assigning a label from an appropriately matching element in the div.
